### PR TITLE
Update VM size to Standard_E4_v3

### PR DESCRIPTION
**Pull request by guyhay**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@guyhay - it looks like E-series is not supported for HDI in FF (I'm guessing)... is that a bug or do we need to use different param files for different clouds?

**Comment by guyhay** [comment]
It should be, but it new.  Let me go investigate further.

Guy

**Comment by guyhay** [comment]
Thanks for picking this up.  Is the list of VMs supported in our RP or somewhere else?

Guy

**Review by JustineNicole2020 — APPROVED** [review]

**Comment by bmoore-msft** [comment]
@guyhay - who are you directing the question to?

**Comment by guyhay** [comment]
More details here.  @Deepali Shrivastava<mailto:Deepali.Shrivastava@microsoft.com> the update to this template is failing in US Gov Virginia because Standard_E4_v3 isn't yet available.  Details  From Pipelines - Run 43426 logs (azure.com)<https://dev.azure.com/azurequickstarts/azure-quickstart-templates/_build/results?buildId=43426&view=logs&j=f4a2bb40-7e70-5f50-d570-7d7c5c8ca123&t=c90a1f86-96f3-5f4d-303e-ef98c0c57aed&l=93>

[Text  Description automatically generated]

Error: 'VM size 'standard_e4_v3' provided in the request is not allowed. Valid values are: STANDARD_D12,STANDARD_D12_V2,STANDARD_D13,STANDARD_D13_V2,STANDARD_D14,STANDARD_D14_V2,STANDARD_D3,STANDARD_D3_V2,STANDARD_D4,STANDARD_D4_V2,STANDARD_D5_V2,STANDARD_DS12_V2,STANDARD_DS13_V2,STANDARD_DS14_V2,STANDARD_DS3_V2,STANDARD_DS4_V2,STANDARD_DS5_V2,STANDARD_A2_V2,STANDARD_A3,STANDARD_A4,STANDARD_A5,STANDARD_A6,STANDARD_A7' (Code:InvalidDocumentErrorCode)

What's our plan, and what is the work to deploy Standard_E4_v3 there?  Without this support we are blocked from updating and quickstart templates.

Guy

From: Guy Haycock
Sent: Sunday, January 10, 2021 16:14
To: Azure/azure-quickstart-templates <reply@reply.github.com>; Azure/azure-quickstart-templates <azure-quickstart-templates@noreply.github.com>
Cc: Mention <mention@noreply.github.com>; Mimi Gentz <mimig@microsoft.com>; Deepali Shrivastava <Deepali.Shrivastava@microsoft.com>
Subject: RE: [Azure/azure-quickstart-templates] Update VM size to Standard_E4_v3 (#8647)

Thanks for picking this up.  Is the list of VMs supported in our RP or somewhere else?

Guy

**Comment by bmoore-msft** [comment]
@guyhay - if you want to unblock, you can add a separate param file for FF that uses the old size...  It's not idea, since you have to remember to come back and remove it when parity shows up, but it is an option if you want it...

azuredeploy.parameters.us.json

**Comment by guyhay** [comment]
Thanks.  I'm going down the path of getting Ev3 into FF.

Guy

File: 101-hdinsight-linux-add-edge-node/README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@
...
-[![Deploy To Azure US Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-hdinsight-linux-add-edge-node%2Fazuredeploy.json)  
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-hdinsight-linux-add-edge-node%2Fazuredeploy.json)
+[![Deploy To Azure US Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-hdinsight-linux-add-edge-node%2Fazuredeploy.json)
```
Comment (bmoore-msft) [inline 101-hdinsight-linux-add-edge-node/README.md:13]: this one should be .us

Comment (guyhay) [reply 101-hdinsight-linux-add-edge-node/README.md]: updating

**Pull request closed without merging**